### PR TITLE
[Swiftlint] Moar swiftlint fixes

### PR DESCRIPTION
**Pull request by ajsecord**
Note a very few of these were actually normal compiler warnings, not sure how they snuck in.

### CODE REVIEW

File: components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift [hunk 1]
```diff
@@ -89,12 +77,30 @@ class ButtonsSimpleExampleSwiftViewController: UIViewController {
     // Dispose of any resources that can be recreated.
```
Comment (willlarche) [inline components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift]: didReceiveMemoryWarning() should be removed from examples.

File: components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift [hunk 2]
```diff
@@ -21,6 +21,10 @@ class ButtonsSimpleExampleSwiftViewController: UIViewController {
 
   let floatingButtonPlusDimension = CGFloat(24)
 
+  class func catalogBreadcrumbs() -> [String] {
+    return ["Buttons", "Buttons (Swift)"]
+  }
+
   override func viewDidLoad() {
     super.viewDidLoad()
```
Comment (willlarche) [inline components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift:30]: Remove 'self' before view in this method.

Comment (willlarche) [reply components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift:30]: Also remove 'UIColor' before .[color]

File: components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift [hunk 3]
```diff
@@ -89,12 +77,30 @@ class ButtonsSimpleExampleSwiftViewController: UIViewController {
...
+      relatedBy: .equal,
+      toItem: onView,
+      attribute: .centerY,
+      multiplier: 1.0,
+      constant: 0.0))
   }
 
-  class func catalogBreadcrumbs() -> [String] {
-    return ["Buttons", "Buttons (Swift)"]
+  func tap(_ sender: AnyObject) {
```
Comment (willlarche) [inline components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift]: Change AnyObject to Any for Swift 3

File: components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift [hunk 4]
```diff
@@ -57,6 +57,7 @@ class ButtonsSwiftAndStoryboardController: UIViewController {
     super.init(coder: aDecoder)
```
Comment (willlarche) [inline components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift:57]: This whole example is not actually running properly. May be related to issue #1156 . Maybe should have its own ticket. (IBOutlets aren't hooked up.)

File: components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift [hunk 5]
```diff
@@ -57,6 +57,7 @@ class ButtonsSwiftAndStoryboardController: UIViewController {
...
   }
 
+  // swiftlint:disable function_body_length
   override func viewDidLoad() {
     super.viewDidLoad()
```
Comment (willlarche) [inline components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift:62]: Remove selfs in this method.

File: components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift [hunk 6]
```diff
@@ -176,6 +177,7 @@ class ButtonsSwiftAndStoryboardController: UIViewController {
         metrics: nil,
         views: views))
   }
+  // swiftlint:enable function_body_length
 
   @IBAction func tap(_ sender: AnyObject) {
```
Comment (willlarche) [inline components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift]: Change AnyObject to Any

File: demos/Shrine/Shrine/ShrineDetailViewController.swift [hunk 7]
```diff
@@ -51,6 +51,16 @@ class ShrineDetailView: UIScrollView {
       })
     }
 
+    configureTitleLabel(label: label, labelPadding: labelPadding)
```
Comment (willlarche) [inline demos/Shrine/Shrine/ShrineDetailViewController.swift:54]: Remove self from layoutSubviews()

Comment (willlarche) [reply demos/Shrine/Shrine/ShrineDetailViewController.swift:54]: Remove UIColor

Comment (willlarche) [reply demos/Shrine/Shrine/ShrineDetailViewController.swift:54]: Also remove self in viewDidLoad()

File: demos/Shrine/Shrine/ShrineHeaderContentView.swift [hunk 8]
```diff
@@ -92,6 +92,7 @@ class ShrineHeaderContentView: UIView, UIScrollViewDelegate {
     self.addSubview(logoTextImageView)
   }
 
+  // swiftlint:disable function_parameter_count
```
Comment (willlarche) [inline demos/Shrine/Shrine/ShrineHeaderContentView.swift]: Remove self from commonInit()

File: demos/Shrine/Shrine/ShrineHeaderContentView.swift [hunk 9]
```diff
@@ -129,18 +130,34 @@ class ShrineHeaderContentView: UIView, UIScrollViewDelegate {
...
+    addPage(page: thirdPage as! UIView,
+            imageView: imageView3,
+            label: label3,
+            labelDesc: labelDesc3,
+            cyanBox: cyanBox3,
+            imageName: "heels.png",
+            description: "Better \nwearing heels")
   }
 
   override func layoutSubviews() {
```
Comment (willlarche) [inline demos/Shrine/Shrine/ShrineHeaderContentView.swift:91]: Remove self from this method

File: components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift (hunk 2 above)
Comment (ajsecord) [reply components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift:30]: Done.

File: components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift (hunk 1 above)
Comment (ajsecord) [reply components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift]: Done

File: components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift (hunk 3 above)
Comment (ajsecord) [reply components/Buttons/examples/ButtonsSimpleExampleSwiftViewController.swift]: Fixed the ones the compiler would let me.

File: components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift (hunk 4 above)
Comment (ajsecord) [reply components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift:57]: Done

File: components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift (hunk 5 above)
Comment (ajsecord) [reply components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift:62]: Done

File: components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift (hunk 6 above)
Comment (ajsecord) [reply components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift]: Done

File: demos/Shrine/Shrine/ShrineDetailViewController.swift (hunk 7 above)
Comment (ajsecord) [reply demos/Shrine/Shrine/ShrineDetailViewController.swift:54]: Done

File: demos/Shrine/Shrine/ShrineHeaderContentView.swift (hunk 8 above)
Comment (ajsecord) [reply demos/Shrine/Shrine/ShrineHeaderContentView.swift]: Done

File: demos/Shrine/Shrine/ShrineHeaderContentView.swift (hunk 9 above)
Comment (ajsecord) [reply demos/Shrine/Shrine/ShrineHeaderContentView.swift:91]: Done

**Comment by ajsecord** [comment]
I believe all requested changes are done, @willlarche, PTAL.

**Comment by ajsecord** [comment]
Travis CI is lagging severely. Running `scripts/clean_all && scripts/prep_all && scripts/build_all && scripts/test_all` locally was successful.

**Pull request merged**